documentation/src/main/asciidoc/userguide/chapters/pc/BytecodeEnhancement.adoc

```diff
@@ -95,6 +95,10 @@ include::{example-dir-enhancement}/BytecodeEnhancementTest.java[tags=BytecodeEnh
 
 Bytecode-enhanced bi-directional association management makes that first example work by managing the "other side" of a bi-directional association whenever one side is manipulated.
 
+[IMPORTANT]
+Bytecode-enhanced bi-directional association management has been deprecated as of Hibernate v7.1, and will be removed in a future release. Users are encouraged to move away from this functionality and instead adopt
+correct Java usage as outlined in the <<BytecodeEnhancement-dirty-tracking-bidirectional-correct-usage-example, example>> above.
```

Comment:
I'd prefer to aling this wording with your other PR.  Something like -

```suggestion
Hibernate's bi-directional association management bytecode enhancement feature has been deprecated.  Users should instead manage both sides of such associations directly.
```

We should also clean up uses of "bi-directional" which is actually a word itself: bidirectional.  If you want to do that as part of this PR, great.  If not, we can just add it as a follow-up.  I only see one pre-existing use of that semi-misspelling and it is in this file.

Yes, yes, "pre-existing" may also be written without the hyphen but the hyphenated form is the more formal form ;)

Reply:
Well, 2 uses really, but in the same sentence :D

Reply:
Exceptions in languages ... :cry: there for the sole purpose of pissing people off ... :see_no_evil: :laughing: